src/main/java/org/jitsi/jigasi/CallContext.java

```diff
@@ -152,20 +175,36 @@ public CallContext(Object source)
 
     /**
      * The room name of the MUC room that holds JVB conference call.
-     * @return the room name.
+     * Can just room name or the jid (roomname@conference.tenant.domain.net).
+     *
+     * @return the room name or jid.
```

Comment:
This can't be just the room name, right?

Reply:
Yep.